plugins/catalog/src/components/CatalogFilter/CatalogFilter.tsx

```diff
@@ -105,7 +104,11 @@ export const CatalogFilter: React.FC<CatalogFilterProps> = ({
                       {item.label}
                     </Typography>
                   </ListItemText>
-                  {item.count}
+                  {typeof item.count === 'function' ? (
+                    <item.count />
```

Comment:
Interesting, the API is kind of nice though, I wonder if it’s because it’s a property of an object that I knows it’s not a html element. I can create a different variable in the class which holds that with a capital letter instead